Makefile

```diff
@@ -1,44 +1,62 @@
-clean:
+ifeq ($(UNAME),Darwin)
+     SHELL := /opt/local/bin/bash
+     OS_X  := true
+ else ifneq (,$(wildcard /etc/redhat-release))
+     OS_RHEL := true
+ else
+     OS_DEB  := true
+     SHELL := /bin/bash
+ endif 
+.DEFAULT_GOAL := help
+MAKEFLAGS := --jobs=1
+## Available commands:
+help:           ## Show this help.
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+clean:          ## Remove generated files and directories.
 	rm -rf app resources
 
+build: ## Build the site with Hugo and run link check.
 build: clean
 	hugo --minify
-
 	make check-links-ci
 
+build-preview: ## Build the site for preview.
 build-preview: clean
 	hugo \
 	--baseURL $(DEPLOY_PRIME_URL) \
 	--buildDrafts \
 	--buildFuture \
 	--minify
 
-	make check-links-ci
+# Uncomment the line below if you want to run link checking as part of preview
+#	make check-links-ci
```

Comment:
Maybe just delete all this as check-links-ci doesn't seem to work with these options.